tachyon/math/base/groups_unittest.cc

```diff
@@ -3,8 +3,20 @@
 #include "gmock/gmock.h"
 #include "gtest/gtest.h"
 
+#include "tachyon/base/containers/container_util.h"
+#include "tachyon/math/finite_fields/test/gf7.h"
+
 namespace tachyon::math {
 
+namespace {
+
+class GroupsTest : public testing::Test {
+ public:
+  static void SetUpTestSuite() { math::GF7::Init(); }
+};
+
+}  // namespace
```

Comment:
It's helpful to review if you separate non-related parts from the implementation